packages/cxl-ui/src/components/cxl-app-layout.js

```diff
@@ -111,6 +113,21 @@ export class CXLAppLayoutElement extends LitElement {
     `;
   }
 
+  updated(changedProperties) {
+    super.updated(changedProperties);
+
+    if (changedProperties.has('wide')) {
+      if (this.wide) {
+        if (this._previousAsideOpened !== undefined) {
+          this.asideOpened = this._previousAsideOpened;
+        }
+      } else {
+        this._previousAsideOpened = this.asideOpened;
+        this.asideOpened = false;
+      }
+    }
+  }
+
```

Comment:
Are we able to do a much simpler `${this.wide && this.asideOpened}` conditional upgrade directly on `<aside>` element template?

Reply:
I just tried that. Since `wide` would always evaluate to `false` on mobile the drawer won't open.